lockfile_unix.go

```diff
@@ -34,7 +30,7 @@ func getLockFile(path string, ro bool) (Locker, error) {
 		stateMutex: &sync.Mutex{},
 		rwMutex:    &sync.RWMutex{},
 		file:       path,
-		fd:         uintptr(fd),
+		fd:         uintptr(0),
```

Comment:
`0` is a valid file descriptor value, use `-1` please, if possible. Or, at the very least, leave the field uninitialized = default-initialized; it will still be zero, but at least it will not look like a value anything should rely on.

Reply:
Removed the line. Thanks!